test/shopify_app/configuration_test.rb

```diff
@@ -46,12 +50,22 @@ class ConfigurationTest < ActiveSupport::TestCase
     assert ShopifyApp.configuration.has_webhooks?
   end
 
-  test "webhooks_manager_queue_name and scripttags_manager_queue_name are :default if not configured" do
-    assert_equal :default, ShopifyApp.configuration.webhooks_manager_queue_name
-    assert_equal :default, ShopifyApp.configuration.scripttags_manager_queue_name
+  test "webhooks_manager_queue_name and scripttags_manager_queue_name are equal to ActiveJob queue_name if not configured" do
+    Rails.application.config.active_job.queue_name = :'custom-queue-name'
+
+    assert_equal :'custom-queue-name', ShopifyApp.configuration.webhooks_manager_queue_name
+    assert_equal :'custom-queue-name', ShopifyApp.configuration.scripttags_manager_queue_name
+  end
+
+  test "webhooks_manager_queue_name and scripttags_manager_queue_name are nil if not configured and ActiveJob queue_name is nil" do
+    Rails.application.config.active_job.stubs(:queue_name).returns(nil)
```

Comment:
That's not going to describe the real behavior if we set the value to `:default` in the tests.
If we don't define `config.active_job.queue_name` it is going to be nil but it is going to be replaced by the `:default` value by activejob.